base/main_test_bucket_pool.go

```diff
@@ -194,7 +194,10 @@ func (tbp *TestBucketPool) GetTestBucketAndSpec(t testing.TB) (b Bucket, s Bucke
 			FatalfCtx(ctx, "nil TestBucketPool, but not using a Walrus test URL")
 		}
 
-		walrusBucket := walrus.NewBucket(tbpBucketNamePrefix + "walrus_" + GenerateRandomID())
+		walrusBucket, err := walrus.GetBucket(s.Server, DefaultPool, tbpBucketNamePrefix+"walrus_"+GenerateRandomID())
```

Comment:
Does using GetBucket here (with the potential for file-backed walrus buckets) imply that we should be doing extra work in the teardown for file system cleanup?

Reply:
That would be up to the test. If you specify a file using walrus:// the expectation would be that you tidy that up.